pal/system-info/web/system-info.ts

```diff
@@ -200,9 +200,9 @@ class SystemInfo extends EventTarget {
         const supportXR = typeof (navigator as any).xr !== 'undefined';
         // refer https://stackoverflow.com/questions/47879864/how-can-i-check-if-a-browser-supports-webassembly
         const supportWasm = (() => {
-            // iOS 15.4 has some wasm memory issue, can not use wasm for bullet
-            const isiOS15_4 = (this.os === OS.IOS || this.os === OS.OSX) && /(OS 15_4)|(Version\/15.4)/.test(window.navigator.userAgent);
-            if (isiOS15_4) {
+            // safari iOS 15.4 has some wasm memory issue, can not use wasm for bullet
```

Comment:
I think should modify the comment too? As not only has issue on iOS.

Reply:
udpated https://github.com/cocos/cocos-engine/pull/14974/commits/de35e29db68b7871de0d9e093f3b58e29f41fac8